asr-customize-speechhints.ipynb

```diff
@@ -0,0 +1,274 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0cdc3b68-b297-4e27-8fc4-67da496f8bb8",
+   "metadata": {},
+   "source": [
+    "# Creating Grammars for speech hints"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c8940eb9-822a-4d3c-8fdd-23ba7fe78540",
+   "metadata": {},
+   "source": [
+    "This tutorial walks you through the process of creating custom speech hint grammars for use in inverse text normalization.\n",
+    "The primary application of speech hint grammars is to provide specific normalization post Automatic Speech Recognition (ASR).\n",
+    "\n",
+    "## Dependencies\n",
+    "You need to download the [speech hints grammars](https://github.com/nvidia-riva/speech-hints)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "db1d6936-a70d-4362-8540-34b2768d8ff0",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "This tutorial assumes that you are familiar with finite state acceptors and transducers. Familiarity with the [Pynini](https://www.openfst.org/twiki/bin/view/GRM/Pynini) library and Nemo's [WFST Tutorial](https://github.com/NVIDIA/NeMo/blob/main/tutorials/text_processing/WFST_Tutorial.ipynb) is assumed."
```

Comment:
```suggestion
    "This tutorial assumes that you are familiar with finite state acceptors and transducers. Familiarity with the [Pynini](https://www.openfst.org/twiki/bin/view/GRM/Pynini) library and Nemo's [Weighted Finite-State Transducers (WFST) tutorial](https://github.com/NVIDIA/NeMo/blob/main/tutorials/text_processing/WFST_Tutorial.ipynb) is assumed."
```

Reply:
We should always spell out the first usage of an acronym that we are going to refer to throughout the topic. Also, "Tutorial" is not part of the title so we shouldn't treat is as such. It should be lowercase.